tests/commands/test_workshop_mode.py

```
@@ -0,0 +1,111 @@
+    monkeypatch.chdir(temp_dir)
+    # Make .athf a file so mkdir raises OSError — logging becomes unavailable.
+    (temp_dir / ".athf").write_text("not a directory")
+
+    agent = FakeAgent()
+    _apply_workshop_mode(agent, token_cap=256)
+
+    assert agent._call_llm("hello", max_tokens=4096) == "ok"
+    assert agent.calls == [256]
+
+
+def test_run_rejects_zero_token_cap(monkeypatch):
```

Ruff (0.16.0)

[warning] Unused function argument: `monkeypatch`

(ARG001)